hermes-management/build.gradle

```diff
@@ -62,6 +62,16 @@ yarnSetup.dependsOn(nodeSetup)
 yarn.dependsOn(npmSetup)
 
 task buildHermesConsole(type: YarnTask, dependsOn: yarn) {
+    def tasksThatDontRequireConsole = [
```

Comment:
**question:** Every other task than `[integrationTest, slowIntegrationTest, check]` will trigger `buildHermesConsole`?

I'm wondering whether it couldn't be implemented the other way around, that only certain tasks like `build` or `jar` can trigger `buildHermesConsole`

Reply:
> question: Every other task than [integrationTest, slowIntegrationTest, check] will trigger buildHermesConsole?

Yes, it will.

> I'm wondering whether it couldn't be implemented the other way around, so only certain tasks like build or jar would trigger buildHermesConsole.

Yes, it could be. However, I'm not sure if I can easily construct a comprehensive list of tasks that require the console.